src/components/templates/AnyWayGridFilterMenu.tsx

```diff
@@ -0,0 +1,119 @@
+import FormHelperText from '@material-ui/core/FormHelperText';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+import CalendarTodayIcon from '@material-ui/icons/CalendarToday';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+import { Button, Tooltip } from '@material-ui/core';
+interface IProps {
+    
+}
+
+const useStyles = makeStyles(theme => ({
+    formControl: {
+      margin: theme.spacing(1),
+      minWidth: 120,
+    },
+    selectEmpty: {
+      marginTop: theme.spacing(2),
+    },
+  }));
+const AnyWayGridFilterMenu: FunctionComponent<IProps> = ({ children }) => {
+    const classes = useStyles();
+    const [timeBack, setTimeBack] = React.useState(3);
+    const [timeRangeActive,setTimeRangeActive] = React.useState(false)
+    const [selectedDate, setSelectedDate] = React.useState(new Date());
+    const [selectedDateEnd, setSelectedDateEnd] = React.useState(new Date());
+
+    function  handleDateChange (d : any) {
+      setSelectedDate(d);
+      console.log('chosen date', d)
+    };
+    function  handleDateChangeEnd (d : any) {
+      setSelectedDateEnd(d);
+    };
+    const devStyles = {
+        width: '100%',
+        height : '50px',
+        display: 'inline-block'
+        
+    }
+    const handleChange = (event:any) : void => {
+        setTimeBack(event.target.value);
+        
+      };
+
+    const handleTimeRangeClick = (event:any) : void => {
+      setTimeRangeActive(!timeRangeActive);
+      
+    };
+    return  <div style={devStyles} >
+              <span style={{display : "inline-block", width : "100%"}}>
+              <span style={{position: "absolute", marginTop : "-15px"}}>
+                <Button onClick={handleTimeRangeClick}>
+                  <Tooltip title ={'בחר טווח זמנים'}>
+                    <CalendarTodayIcon/>
+                  </Tooltip>
+                </Button>
+                <FormControl className={classes.formControl} style={{marginTop: "-20px",  display : timeRangeActive ? "None" : "inline-block" }}>
+                  <InputLabel id="demo-simple-select-label"> זמן אחורה</InputLabel>
+                  <Select
+                  labelId="demo-simple-select-label"
+                  id="demo-simple-select"
+                  value={timeBack}
+                  onChange={handleChange}
+                  >
+                    <MenuItem value={3}>3 חודשים</MenuItem>
+                    <MenuItem value={6}>חצי שנה</MenuItem>
+                    <MenuItem value={12}>שנה</MenuItem>
+                    <MenuItem value={60}>5 שנים</MenuItem>
+                  </Select>
+                  </FormControl>
+                  </span>
+                  <span style={{display : timeRangeActive ? "inline-block" : "None" , float: "right", marginRight: "5%", marginTop: "-36px"}}>
```

Comment:
Don't use inline style unless absolutely necessary.

Our styling engine, being JS-based, allows us to pick different classes according to a condition.